src/component/felles/menu/Menu.tsx

```diff
@@ -1,51 +1,40 @@
 import styles from './Menu.module.less';
-import { Normaltekst } from 'nav-frontend-typografi';
-import { useValgtVirksomhetStore } from '../../../store/valgt-virksomhet-store';
+import { Normaltekst, Systemtittel } from 'nav-frontend-typografi';
 import React from 'react';
+import cls from 'classnames';
 import { useDataStore } from '../../../store/data-store';
-import { useHistory } from 'react-router-dom';
+import { AlertStripeInfo } from 'nav-frontend-alertstriper';
+import navLogo from './nav-logo.svg';
+import { VirksomhetVelger } from './virksomhet-velger/VirksomhetVelger';
 
 export const Menu = () => {
```

Comment:
Kan denne hete for eksempel BannerMeny, eller noe annet som er litt mer konkret

Reply:
Kan hende at meny ikke er helt riktig ord enda siden det ikke er lenker osv der. Kanskje bedre med kun Banner enn så lenge, men hvis det blir lenker og navigasjon der så burde vi bytte tilbake til meny igjen